lib/DOMException-impl.js

```diff
@@ -20,3 +23,15 @@ exports.init = impl => {
     Error.captureStackTrace(wrapper, wrapper.constructor);
   }
 };
+
+// Special install function to make the DOMException inherit from Error.
```

Comment:
The circular dependency here seems pretty weird, so I'd rather move this into webidl2js-wrapper.js. (Then maybe delete the inserted whitespace lines so that the diff is just `globalObject, `